### PR TITLE
Doc update about handeye calibration switching from panel to display type in rviz

### PR DESCRIPTION
**Pull request by mfkenson**
### Description

Doc update about handeye calibration switching from panel to display type in rviz.

### Checklist
- [ ] **Required by CI**: Code is auto formatted using [clang-format](http://moveit.ros.org/documentation/contributing/code)
- [ ] While waiting for someone to review your request, please consider reviewing [another open pull request](https://github.com/ros-planning/moveit/pulls) to support the maintainers

[//]: # "You can expect a response from a maintainer within 7 days. If you haven't heard anything by then, feel free to ping the thread. Thank you!"

### CODE REVIEW

File: doc/hand_eye_calibration/hand_eye_calibration_tutorial.rst [hunk 1]
```diff
@@ -35,15 +35,15 @@ Then, make sure you have the appropriate dependencies and build the package::
 Launch RViz and Load Calibration Plugin
 ---------------------------------------
 Launch the appropriate MoveIt demo for your robot. For instance, ``roslaunch panda_moveit_config demo.launch``.
-In the RViz "Panels" menu, choose "Add New Panel":
+In the RViz "Displays" panel, click "Add":
 
-.. image:: images/choose_new_panel.png
```
Comment (vatanaksoytezer) [inline doc/hand_eye_calibration/hand_eye_calibration_tutorial.rst:40]: Could you please delete this file?

File: doc/hand_eye_calibration/hand_eye_calibration_tutorial.rst [hunk 2]
```diff
@@ -35,15 +35,15 @@ Then, make sure you have the appropriate dependencies and build the package::
...
+.. image:: images/choose_add_display.png
 
-Then, select the "HandEyeCalibration" panel type:
+Then, select the "HandEyeCalibration" display type:
 
-.. image:: images/add_handeye_panel.png
```
Comment (vatanaksoytezer) [inline doc/hand_eye_calibration/hand_eye_calibration_tutorial.rst:44]: and this.

**Review by vatanaksoytezer — COMMENTED** [review]
Thanks a lot for the contribution @mfkenson! Do you mind also deleting the old png files, so we don't have them staying around.

File: doc/hand_eye_calibration/hand_eye_calibration_tutorial.rst (hunk 1 above)
Comment (mfkenson) [reply doc/hand_eye_calibration/hand_eye_calibration_tutorial.rst:40]: sure.

File: doc/hand_eye_calibration/hand_eye_calibration_tutorial.rst (hunk 2 above)
Comment (mfkenson) [reply doc/hand_eye_calibration/hand_eye_calibration_tutorial.rst:44]: done.

**Comment by mfkenson** [comment]
ready for review. @vatanaksoytezer 

Latest commit in this PR deleted these two files.
* images/choose_new_panel.png
* images/add_handeye_panel.png

**Review by vatanaksoytezer — APPROVED** [review]

**Pull request merged**